__tests__/api.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api", () => {
+    it("will return 200 status code", () => {
+        return request(app)
+        .get("/api")
+        .expect(200);
+    })
+    it("will return 200 and the JSON of all available endpoints", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
```

Comment:
Because we are asserting in here that we are getting the 200 status code anyway, both tests are not necessary (you could either keep just the second test as it is and get rid of the first one, or remove line 19 and keep both tests)